main.css

```diff
@@ -0,0 +1,86 @@
+html, body {
+    height: 100%;
+    margin: 0;
+    font-family: 'Roboto Mono', monospace;
+}
```

Comment:
Page is scrollbar on x-axis (might be caused by refactoring of  existing styles for header/footer/content blocks in main.css)

Reply:
Hmm I'll look into it

Reply:
updated ✅